src/helper/helper.go

```
@@ -49,27 +49,190 @@ type versionResponse struct {
+
+func (f *interactiveFormatter) writeText(text string) {
+	for _, ch := range text {
+		char := string(ch)
+		if !f.inXMLTag {
+			if ch == '<' {
+				f.inXMLTag = true
+				f.xmlBuffer.Reset()
+				f.xmlBuffer.WriteRune(ch)
+				continue
+			}
+			f.streamFormatter.writeChar(char)
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "streamFormatter" from selector

(staticcheck)